Turcica.csl

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl">  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>turcica</title>
+    <id>http://www.zotero.org/styles/turcica</id>
+    <link rel="self" href="http://www.zotero.org/styles/revue-turcica"/>
```

Comment:
This still has the wrong name. Remove "revue-"
Also, your filename needs to be changed to "turcica.csl" (no caps)